Allowed .NET to use TLS 1.1 and 1.2

akuryan opened the pull request:
Now it is possible to select TLS version for Azure web apps. Today, when I was trying manual renew of a certificate, I was not able to do it, as my web app uses only TLS 1.2 connection - so, I was obliged to push TLS to 1.0, renew certificate manually and push it back to 1.2

sjkp commented:
What was your test setup, in order for this change to be mandatory? 
If I enable TLS 1.2, the acme challenge is still done via http, so I don't really need the changes you added, and the old version works just fine.

akuryan commented:
It was a renew of certificate, not new installation.
I have an Azure Web app, using TLS 1.2 only, with Let's Encrypt certificate, provided via this extension, which was about to expire. I requested renewal for certificate and received YSOD, which was saying that HTTPS connection was forcibly closed. So, I moved to TLS 1.0, renewed certificate and moved up to 1.2
